### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SkiingIsFun123**
Improved the readability of the readme.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -374,7 +374,7 @@ run SwiftLint from. The following parameters can be configured:
 Rule inclusion:
 
 * `disabled_rules`: Disable rules from the default enabled set.
-* `opt_in_rules`: Enable rules not from the default set.
+* `opt_in_rules`: Enable rules, not from the default setting.
```
Comment (SimplyDanny) [inline README.md:377]: I'd like
```suggestion
* `opt_in_rules`: Enable rules that are not part of the default set.
```
more. What do you think?

File: README.md [hunk 2]
```diff
@@ -634,7 +634,7 @@ specifications of nested configurations are getting ignored because there's no s
 If one (or more) SwiftLint file(s) are explicitly specified via the `--config` parameter,
 that configuration will be treated as an override, no matter whether there exist
 other `.swiftlint.yml` files somewhere within the directory. **So if you want to use
-use nested configurations, you can't use the `-- config` parameter.**
+ nested configurations, you can't use the `-- config` parameter.**
```
Comment (SimplyDanny) [inline README.md:637]: Could you also remove the space in the option:
```suggestion
 nested configurations, you can't use the `--config` parameter.**
```

**Review by SimplyDanny — COMMENTED** [review]
Thank you for the tweaks!

**Comment by SkiingIsFun123** [comment]
No problem! I implemented the changes

**Comment by SimplyDanny** [comment]
> No problem! I implemented the changes

Thank you!

**Pull request merged**